browser/ui/webui/settings/brave_adblock_handler.cc

```diff
@@ -0,0 +1,304 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/webui/settings/brave_adblock_handler.h"
+
+#include <memory>
+#include <string>
+#include <utility>
+
+#include "base/bind.h"
+#include "base/json/values_util.h"
+#include "base/values.h"
+#include "brave/browser/brave_browser_process.h"
+#include "brave/components/brave_adblock/resources/grit/brave_adblock_generated_map.h"
+#include "brave/components/brave_shields/browser/ad_block_custom_filters_provider.h"
+#include "brave/components/brave_shields/browser/ad_block_regional_service_manager.h"
+#include "brave/components/brave_shields/browser/ad_block_service.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/browser/content_settings/host_content_settings_map_factory.h"
+#include "chrome/browser/profiles/profile.h"
+#include "content/public/browser/web_ui.h"
+#include "ui/base/l10n/time_format.h"
+
+#if BUILDFLAG(IS_ANDROID)
+#include "chrome/browser/android/tab_web_contents_delegate_android.h"
+#include "content/public/browser/web_contents.h"
+#else
+#include "chrome/browser/ui/browser_finder.h"
+#include "chrome/browser/ui/browser_navigator_params.h"
+#include "chrome/browser/ui/singleton_tabs.h"
+#endif
```

Comment:
The settings page won't ever render on Android, so we can remove the conditional

Reply:
removed